themes/bootstrap3/templates/RecordDriver/DefaultRecord/link-generic.phtml

```diff
@@ -0,0 +1,13 @@
+<?php
+  $searchBackendId = $this->driver->getSearchBackendIdentifier();
+  $options = $this->searchOptions($searchBackendId);
+  $query = [
+    'type' => $this->type,
+    'lookfor' => ($this->quoteQuery ?? true) ? '"' . $this->lookfor . '"' : $this->lookfor,
```

Comment:
I see that this `quoteQuery` logic accurately reproduces the current behavior, so it's reasonable to maintain the two options.  But I'm wondering whether we actually know that the "no quotes" option is needed for some types/backends, or if it may just be an oversight that was reproduced across backends and now we're supporting it as a feature.  In case someone picks up that thread in the future.

Reply:
In some cases, I think it's definitely intentional -- e.g. in some backends, author search is unquoted to support inconsistent first/last name order. In other cases it seems inconsistently applied, and it's possible as you say that it's simply an oversight or error. For this PR, my intent was to keep the functionality the same to reduce potential future confusion... but as you say, it may be worth revisiting in the future.

Reply:
Makes sense and good to know for the future.